lib/options.js

```diff
@@ -66,9 +66,9 @@ module.exports = {
         default: 'example',
         availableOptions: ['example', 'schema'],
         description: 'Determines how response parameters (query parameters, path parameters, headers,' +
-         'or the request body) should be generated. Setting this to schema will cause the importer to' +
-         'use the parameter\'s schema as an indicator; `example` will cause the example (if provided)' +
-         'to be picked up.',
+         ' or the request body) should be generated. Setting this to schema will cause the importer to' +
+         ' use the parameter\'s schema as an indicator; `example` will cause the example (if provided)' +
```

Comment:
let's also markdown-link schema and example to https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md#schemaObject and https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md#exampleObject

Reply:
Done